crates/symbolicator/src/services/cacher.rs

```diff
@@ -421,13 +421,31 @@ impl<T: CacheItemRequest> Cacher<T> {
 
         // Run the computation and wrap the result in Arcs to make them clonable.
         let channel = async move {
+            // only start an independent transaction if this is a "background" task,
+            // otherwise it will not "outlive" its parent span, so attach it to the parent transaction.
+            let transaction = if is_refresh {
+                let span = sentry::configure_scope(|scope| scope.get_span());
+                let ctx = sentry::TransactionContext::continue_from_span(
+                    "Lazy Cache Computation",
+                    "spawn_computation",
+                    span,
+                );
+                let transaction = sentry::start_transaction(ctx);
+                sentry::configure_scope(|scope| scope.set_span(Some(transaction.clone().into())));
```

Comment:
not for here, but what's the logic for `.set_span` to take an `Option` rather than the span/transaction directly?  Seems weird

Reply:
Well, you are supposed to unset the "active" span somehow. The unified API unfortunately does not really make this very clear, meh.

Reply:
I we have some creative license wrt to the unified API, for rust I expect more `fn set_span(&self, span: impl TracingSpan)` and `fn detach_span(&self)` or something